vowpalwabbit/array_parameters_dense.h

```diff
@@ -86,44 +123,61 @@ class dense_parameters
     _seeded = true;
   }
 
-  inline weight& strided_index(size_t index) { return operator[](index << _stride_shift); }
+  inline weight& strided_index(size_t index)
+  {
+    return operator[](index << _stride_shift);
+  }
 
-  template<class R, class T> void set_default(R& info)
+  template <class R, class T>
+  void set_default(R& info)
   {
     iterator iter = begin();
-    for (size_t i = 0; iter != end(); ++iter, i += stride())
-      T::func(*iter, info, iter.index());
+    for (size_t i = 0; iter != end(); ++iter, i += stride()) T::func(*iter, info, iter.index());
   }
 
-  template<class T> void set_default()
+  template <class T>
+  void set_default()
   {
     iterator iter = begin();
-    for (size_t i = 0; iter != end(); ++iter, i += stride())
-      T::func(*iter, iter.index());
+    for (size_t i = 0; iter != end(); ++iter, i += stride()) T::func(*iter, iter.index());
   }
 
   void set_zero(size_t offset)
   {
-    for (iterator iter = begin(); iter != end(); ++iter)
-      (&(*iter))[offset] = 0;
+    for (iterator iter = begin(); iter != end(); ++iter) (&(*iter))[offset] = 0;
   }
 
-  uint64_t mask()	const { return _weight_mask; }
+  uint64_t mask() const
+  {
+    return _weight_mask;
+  }
 
-  uint64_t seeded() const { return _seeded; }
+  uint64_t seeded() const
+  {
+    return _seeded;
+  }
 
-  uint32_t stride() const { return 1 << _stride_shift; }
+  uint32_t stride() const
+  {
+    return 1 << _stride_shift;
+  }
 
-  uint32_t stride_shift() const { return _stride_shift; }
+  uint32_t stride_shift() const
+  {
+    return _stride_shift;
+  }
 
-  void stride_shift(uint32_t stride_shift) { _stride_shift = stride_shift; }
+  void stride_shift(uint32_t stride_shift)
+  {
+    _stride_shift = stride_shift;
+  }
 
 #ifndef _WIN32
 #ifndef DISABLE_SHARED_WEIGHTS
   void share(size_t length)
   {
-    float* shared_weights = (float*)mmap(0, (length << _stride_shift) * sizeof(float),
-      PROT_READ | PROT_WRITE, MAP_SHARED | MAP_ANONYMOUS, -1, 0);
+    float* shared_weights = (float*)mmap(0, (length << _stride_shift) * sizeof(float), PROT_READ | PROT_WRITE,
+                                         MAP_SHARED | MAP_ANONYMOUS, -1, 0);
```

Comment:
wrapped lines should not be aligned, just indented

Reply:
I'm not sure it can be verbatim the same. The previous version start a new unaligned line for the third argument. Using `AlignAfterOpenBracket: DontAlign` for some reason forces it to start a new indented line with the first argument. 

The other alternatives for the option I used (`AlignAfterOpenBracket`) to get indentation instead of alignment are `Align` (the intermediate change we don't want) and `AlwaysBreak` (probably will look the same as now).

Reply:
It probably forces it because the line is too long, not sure. `AlignAfterOpenBracket: DontAlign` seems best?